source/_integrations/alexa.intent.markdown

```diff
@@ -36,12 +30,89 @@ To get started with Alexa skills:
   - Name: Home Assistant
   - Invocation name: home assistant (or be creative, up to you)
   - Version: 1.0
-  - Endpoint:
-    - https
-    - `https://YOUR_HOST/api/alexa?api_password=YOUR_API_PASSWORD`
+  - Endpoint: This will be the ARN for the Lambda Function you will create next.
 
 You can use this [specially sized Home Assistant logo][large-icon] as the large icon and [this one][small-icon] as the small one.
 
+### Create Your Lambda Function
+
+The Alexa Custom skill will trigger a AWS Lambda function to process the request, we will write a small piece of code hosted as a Lambda function to basically redirect the request to your Home Assistant instance, then the Alexa integration in Home Assistant will process the request and send back the response. Your Lambda function will deliver the response back to Alexa.
+
+OK, let's go. You first need sign in your [AWS console](https://console.aws.amazon.com/), if you don't have an AWS account yet, you can create a new user [here](https://aws.amazon.com/free/) with 12-month free tier benefit. You don't need worry the cost if your account has already passed the first 12 months, AWS provides up to 1 million Lambda requests, 1GB of outbound data and unlimited inbound data for free every month for all users. See [Lambda pricing](https://aws.amazon.com/lambda/pricing/) for details.
+
+#### Create an IAM Role for Lambda 
```

Comment:
Instead of these instructions, someone should just create a GitHub repo with a serverless.com implementation. Then it becomes as easy as `npm install && aws login && npm run deploy`